lib/workspace/commands/service.rb

```diff
@@ -34,6 +34,7 @@ def new(service)
 
       desc 'start SERVICE', 'Starts up the specified service'
       option :build, type: :boolean
+      option :d, type: :boolean
```

Comment:
We should make this the alias, similar to how `image` and `platform` are for the above command.
```suggestion
      option :detach, :type => :boolean, :aliases => :d, desc: 'provide documentation here'
```